src/tests/api/BavUnhappyPath.test.ts

```diff
@@ -0,0 +1,34 @@
+import bavStubPayload from "../data/exampleStubPayload.json";
+import { constants } from "../utils/ApiConstants";
```

Comment:
Again, let's get rid of the unused imports. 

Only bavStubPayload, stubStartPost and sessionPost needed